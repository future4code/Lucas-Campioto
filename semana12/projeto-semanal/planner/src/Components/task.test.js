import React from 'react'
import renderer from "react-test-renderer";
import Task from './task';

describe ('teste componente task', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Task></Task>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})