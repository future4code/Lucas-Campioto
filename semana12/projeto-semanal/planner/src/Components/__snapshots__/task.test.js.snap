// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste componente task renders correctly 1`] = `
<div>
  <h3 />
</div>
`;
